api/room_resource/schema.py

```diff
@@ -107,6 +107,28 @@ def mutate(self, info, resource_id, room_id, **kwargs):
         return UpdateAssignedResource(room_resource=room_resource)
 
 
+class DeleteAssignedResource(graphene.Mutation):
+    """
+        Delete assigned resource in a room
+    """
+    class Arguments:
+        resource_id = graphene.Int(required=True)
+        room_id = graphene.Int(required=True)
+    room_resource = graphene.Field(RoomResource)
+
+    @Auth.user_roles('Admin')
+    def mutate(self, info, resource_id, room_id):
+        query = RoomResource.get_query(info)
+        assigned_resource = query.filter(
+            RoomResourceModel.room_id == room_id,
+            RoomResourceModel.resource_id == resource_id).first()
+        if not assigned_resource:
```

Comment:
Good work @jesseinit however, wouldn't be better to do room_id check then a resource_id check separately so as you can raise a clear and specific error. In the current implementation, it always raises   `The resource has not been assigned to the specified room ` when either the room_id or resource_id does not exist.

Reply:
Nice catch. I'll update accordingly.